examples/bond/scripts/run/buy.js

```diff
@@ -0,0 +1,71 @@
+const {
+  executeTransaction, convert
+} = require('@algo-builder/algob');
+const { types } = require('@algo-builder/web');
+
+async function run (runtimeEnv, deployer) {
+  const masterAccount = deployer.accountsByName.get('master-account');
+  const creatorAccount = deployer.accountsByName.get('john');
+  const buyerAccount = deployer.accountsByName.get('bob');
+  const storeManagerAccount = deployer.accountsByName.get('alice');
+
+  const algoTxnParams = {
+    type: types.TransactionType.TransferAlgo,
+    sign: types.SignType.SecretKey,
+    fromAccount: masterAccount,
+    toAccountAddr: buyerAccount.addr,
+    amountMicroAlgos: 200000000,
+    payFlags: {}
+  };
+  await executeTransaction(deployer, algoTxnParams);
+
+  const appInfo = deployer.getApp('bond-dapp-stateful.py', 'bond-dapp-clear.py');
+  const scInitParam = {
+    TMPL_APPLICATION_ID: appInfo.appID,
+    TMPL_OWNER: creatorAccount.addr,
+    TMPL_STORE_MANAGER: storeManagerAccount.addr
+  };
+  const issuerLsig = await deployer.loadLogic('issuer-lsig.py', scInitParam);
+  const asaInfo = deployer.getASAInfo('bond-token');
+  await deployer.optInAcountToASA(asaInfo.assetIndex, 'bob', { totalFee: 1000 });
+
+  const algoAmount = 10 * 1000 + 1000;
+
+  const groupTx = [
+    // Algo transfer from buyer to issuer
+    {
+      type: types.TransactionType.TransferAlgo,
+      sign: types.SignType.SecretKey,
+      fromAccount: buyerAccount,
+      toAccountAddr: issuerLsig.address(),
+      amountMicroAlgos: algoAmount,
+      payFlags: {}
+    },
+    // Bond token transfer from issuer's address
+    {
+      type: types.TransactionType.TransferAsset,
+      sign: types.SignType.LogicSignature,
+      fromAccountAddr: issuerLsig.address(),
+      lsig: issuerLsig,
+      toAccountAddr: buyerAccount.addr,
+      amount: 10,
+      assetID: asaInfo.assetIndex,
+      payFlags: { totalFee: 1000 }
+    },
+    // call to bond-dapp
+    {
+      type: types.TransactionType.CallNoOpSSC,
+      sign: types.SignType.SecretKey,
+      fromAccount: buyerAccount,
+      appID: appInfo.appID,
+      payFlags: {},
+      appArgs: ['str:buy']
+    }
+  ];
+
+  console.log('Buying tokens!');
+  await executeTransaction(deployer, groupTx);
+  console.log('Tokens Bought!');
+}
+
```

Comment:
let's update the _run_ scripts to have the following use case (after deploying the contract):
+ in epoch_0 alice buys 10 bonds
+ in epoch 0 alice sells 2 bonds to charlie (new account) for 2020 ALGO (in a group transaction)
+ manager starts epoch 1 (create dex)
+ alice redeems her bonds
+ alice buys 4 more bonds (so she will have 12 bonds in total)
+ manager starts epoch 2 (create dex)
+ alice redeems her bonds
+ charlie redeems her bonds_1 and then bonds_2 (so going through the previous dex)
+ at maturity (let's change 1000 seconds to 240 seconds == 4 min) manager creates and funds buyback and both alice and charlie exit  all their tokens (12 and 2 respectively).

Please describe this scenario in the README. We should have the following files:
+ deploy.js
+ run/epoch0.js
+ run/epoch1.js
+ run/exit.js
and maybe some common files to reuse in the run scripts.

In test we should also add this scenario and make all checks (total atoms received etc...)

Reply:
done, describing in readme now